R/assess_has_examples.R

```diff
@@ -0,0 +1,36 @@
+#' Assess a package for the presence of example or usage fields in function documentation
+#'
+#' @eval roxygen_assess_family(
+#'   "has_examples",
+#'   "an integer value indicating the number of discovered files with examples")
+#'
+#' @export
+assess_has_examples <- function(x, ...) {
+  UseMethod("assess_has_examples")
+}
+
+# assign a friendly name for examples column
+attributes(assess_has_examples)$column_name <- "has_examples"
+attributes(assess_has_examples)$label <- "number of discovered function files with examples"
+
+#' @export
+assess_has_examples.pkg_ref <- function(x, ...) {
+  pkg_metric_eval(class = "pkg_metric_has_examples", {
+    length(x$examples)
```

Comment:
Would a better assessment be to find the proportion of objects that have examples rather than the number?

Reply:
Good idea. I'll adjust that.